docs/_attic/glossary_entries.py

```diff
@@ -514,6 +521,13 @@
 	pronunciation='', 
 	seealso="[workflow]")
 
+TRS = GlossEntry("TRS",
+        acronym_full="Tool Registry Service",
+        definition="A standardized [API], created by the [GA4GH] Cloud Work Stream, that provides portable access to a registry of tools, workflows, and associated files.  Every resource in a TRS registry has a public ID that can be used to retrieve it.  Dockstore `provides <https://dockstore.org/api/static/swagger-ui/index.html#/GA4GH>`__ a TRS interface to its database.",
```

Comment:
The link is pointing to the TRS beta implementation; it should be https://dockstore.org/api/static/swagger-ui/index.html#/GA4GHV20

Arguable nit: "...to its database", seems like an implementation detail, unless you're using database in a very general sense, but I don't have a better suggestion off-hand, so you don't have to change it if you like it.

Reply:
Link fixed.  Re "to its database", yes, I was using that generically, rather than as a reference to PostgresQL or whatever.  It didn't add much, so I deleted it, thus avoiding the need to consider whether it's common usage, should be replaced, etc.